src/scripts/sync_subclassof_collate_self_parentage.py

```diff
@@ -0,0 +1,84 @@
+"""Sync SubClass: Report for Mondo classes having themselves as parents in the "added" template (legal proxy merges)."""
```

Comment:
## Refactor
When I made this report, I did it quick and dirty because I wasn't sure it'd be valued as a permanent output. Nico likes it, but in order to keep it, a refactor is necessary. I think if Nico realized this, maybe he wouldn't have suggested keeping it (as a 'silent' report (e.g. put in `tmp/` and don't throw an error). But I didn't have the foresight to think that Nico would suggest this, and I didn't realize until I'm halfway through this refactor that it actually takes a couple hours. But I'm more than halfway through now.

Reply:
The refactor is now done.